TopPlaces/TopPlaces/Model/Flickr/FlickrPlaceInfo.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2016 Lightricks. All rights reserved.
+// Created by Gennadi Iosad.
+
+#import "PlacesPhotosProvider.h"
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FlickrPlaceInfo : NSObject<PlaceInfo>
+
+- (instancetype)init NS_UNAVAILABLE;
+
+/// Initialize the instance with the \c dict recieved from the Flickr service
+- (instancetype)initWithDictionary:(NSDictionary*)dict NS_DESIGNATED_INITIALIZER;
+
+/// Returns the URL to download the list of the photos from the place,
+/// the list length is bounded by \c maxLength
+- (NSURL*)urlOfPhotoInfoArrayWithMaxLength:(NSUInteger)maxLength;
+
+/// Compares to another instance of PlaceInfo conforming object
+- (NSComparisonResult)compare:(id<PlaceInfo>)other;
```

Comment:
Since this is not trivial you should document which value is returned in different scenarios.